sbin/travis.sh

```diff
@@ -45,6 +51,8 @@ Bootstrap() {
 
         echo "download UnitTesting tag: $UNITTESTING_TAG"
         git clone --quiet --depth 1 --branch $UNITTESTING_TAG "$UT_URL" "$UT_PATH"
+        git -C "$UT_PATH" rev-parse HEAD
```

Comment:
why we need this?

Reply:
When you checkout a branch rather than a tag it doesn't display the commit hash that is checked out. Knowing the commit hash is very useful for debugging.